module/move/wca/src/ca/verifier/verifier.rs

```diff
@@ -4,9 +4,10 @@ pub( crate ) mod private
 
   use ca::grammar::command::ValueDescription;
   // use former::Former;
-  use std::collections::HashMap;
+  use std::collections::{ BTreeMap, HashMap };
   use wtools::{ error, error::Result, err };
   use ca::help::private::{ HelpGeneratorOptions, LevelOfDetail, generate_help_content };
+  use crate::ca::grammar::dictionary::private::CommandName;
```

Comment:
This seems to be not in line with the [convention](https://github.com/obox-systems/conventions/blob/master/code_style.md#importing-local-entities)